ReplaySample/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+
+    defaultConfig {
+        applicationId "com.foresee.ReplaySample"
```

Comment:
With the new naming conventions, this should be `com.verint.xm.DbaSample`, and the app itself should be renamed to match, i.e. all instances of replay should be renamed